Makefile

```
@@ -22,6 +22,7 @@ K_LIB := $(K_RELEASE)/lib/kframework
 export K_RELEASE
 
 LIBRARY_PATH       := $(LOCAL_LIB)
+K_INCLUDE_PATH     := $(BUILD_LOCAL)/include/kframework/
```

```suggestion
K_INCLUDE_PATH     += :$(BUILD_LOCAL)/include/kframework/
```

To allow users to set their own `K_INCLUDE_PATH`, and just extend that one.

---

@ehildenb @dwightguth Currently, we use the variable `K_INCLUDE_PATH` as an argument to `kompile` on like 146 which says:

```
KOMPILE_OPTS += --hook-namespaces "$(HOOK_NAMESPACES)" -I $(K_INCLUDE_PATH)
```

However, it seems that K does not support the colon-delimited path inclusion convention. At least, after this change was integrated, the build seems to be broken. Is there a way to support this correctly, or did I make a mistake?

---

Ok, if we cannot use the `:` convention as expected, please just call it something slightly different, like `K_LOCAL_INCLUDE_PATH`.

---

Done.